integration-test/1983-china-south-sea.py

```diff
@@ -0,0 +1,32 @@
+# -*- encoding: utf-8 -*-
+from . import FixtureTest
+
+
+class NaturalEarth(FixtureTest):
+    def test_claim_boundary(self):
+        import dsl
+
+        z, x, y = 1, 0, 0
+
+        self.generate_fixtures(
+            dsl.way(1, dsl.tile_diagonal_dexter(z, x, y), {
+                'featurecla': 'Claim boundary',
```

Comment:
Change **featurecla** to be `Unrecognized` (data will also change upstream in NE).

Reply:
> Change **featurecla** to be `Unrecognized` (data will also change upstream in NE).

changed in the next commit, this time the kind is "unrecognized country"